Source/IntegrationTests/TestFiles/LitTests/LitTest/dafny0/TypeInferenceRefresh.dfy

```diff
@@ -963,6 +963,38 @@ module StaticReceivers {
 }
 
 
+/****************************************************************************************
```

Comment:
Does this need to be resolved before merging the PR?

Reply:
Thanks. I cleaned this up, but am leaving two of these to-do problems in the test file. I'm looking into one of them now (but as a separate PR, once I resolve it).